src/EnergyPlus/BranchInputManager.cc

```diff
@@ -2977,20 +2965,25 @@ namespace BranchInputManager {
 
         BranchNodes.allocate(NumOfBranches);
 
-        ObjexxFCL::gio::write(OutputFileBNDetails, Format_701) << "! ===============================================================";
-        ObjexxFCL::gio::write(OutputFileBNDetails, Format_700);
+        print(outputFiles.bnd, "{}\n", "! ===============================================================");
+        static constexpr auto Format_700("! <#Branch Lists>,<Number of Branch Lists>");
+        print(outputFiles.bnd, "{}\n", Format_700);
         ObjexxFCL::gio::write(ChrOut, fmtLD) << NumOfBranchLists;
```

Comment:
Why wasn't line 2971 converted?

Reply:
Because it doesn't write to one of the specific output files I was writing to and I missed it.

I'm assuming once I'm done with all of the various output files I'll have another round to go through and remove all of those things.